examples/forms.amp.html

```diff
@@ -0,0 +1,68 @@
+<!doctype html>
+<html ⚡>
+<head>
+    <meta charset="utf-8">
+    <title>Forms Examples in AMP</title>
+    <link rel="canonical" href="amps.html" >
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <style amp-custom>
+        form.amp-form-invalid input:invalid {
+            background-color: #dc4e41;
+        }
+
+        form .invalid-message,
+        /* was invalid but now valid */
+        form.amp-form-invalid:valid .invalid-message,
+        form .now-valid-message {
+            display: none;
+        }
+
+        form.amp-form-invalid .invalid-message {
+            display: block;
+            color: #dc4e41;
+        }
+
+        /* was invalid but now valid */
+        form.amp-form-invalid:valid .now-valid-message {
+            display: block;
+            color: #4CAF50;
+        }
+    </style>
+</head>
+<body>
+<h2>Subscribe to our weekly Newsletter</h2>
+
+<!--
+     AMP would add a class to reflect the validity state of the form after a
+     user tries to submit (.amp-form-valid, .amp-form-invalid) unlike
```

Comment:
Currently submit, left the tries to edit stuff to next PRs specially that those will involve handling events on individual inputs as well.